tools/packager/afu.py

```diff
@@ -92,8 +87,8 @@ def validate(self, packaging=False):
         if self.afu_json == {}:
             return False
         try:
-            jsonschema.validate(self.afu_json, afu_schema)
-        except jsonschema.exceptions.ValidationError as ve:
+            validators.validate(self.afu_json, afu_schema)
+        except exceptions.exceptions.ValidationError as ve:
```

Comment:
As the import is currently written, shouldn't this be `except exceptions.ValidationError as ve:`?

We missed this when we originally added jsonschema in here, but this could have been shortened to `except jsonschema.ValidationError as ve:` before.